components/Carousel.vue

```diff
@@ -0,0 +1,188 @@
+<script>
+import BadgeState from '@/components/BadgeState';
+
+export default {
+  components: { BadgeState },
+  name:       'Carousel',
+  props:      {
+    sliders: {
+      type:    Array,
+      default: null
+    }
+  },
+  data() {
+    return {
+      slider:          this.sliders,
+      activeItemId:    0
+    };
+  },
+
+  computed: {
+
+    trackStyle() {
+      return `transform: translateX(-${ this.activeItemId * 100 / this.slider.length }%); width: calc(${ 60 * this.slider.length }%)`;
```

Comment:
`calc()` is not required as there's no actual computation needed to be handled in the CSS.
Also, to increase readability, please simplify logic by declaring variables, e.g.:

```ts
trackStyle() {
      const x = this.activeItemId * 100 / this.slider.length
      const width = 60 * this.slider.length;

      return `transform: translateX(-${ x }%); width: ${ width }%)`;
}